windows/src/main/csharp/ch/cyberduck/ui/controller/MainController.cs

```diff
@@ -848,7 +856,7 @@ private void InitializeTransfers()
                 _bc.Invoke(() =>
                 {
                     transfersSemaphore.Wait();
-                    TransferController.Instance.View.Show();
+                    TransferController.Instance.ShowWindow();
```

Comment:
View is lazily created if required. Use instance method which handles everything (including dispatching to WPF secondary thread).